### PR TITLE
Advertise our other companies-house gem

### PR DESCRIPTION
**Pull request by twhaples**
We have a gem for the new beta JSON-based service which is similarly named and also may be relevant to people who locate this gem (especially as it does not cost money). Advertise its existence here to reduce confusion and to be helpful.

### CODE REVIEW

**Comment by Baltoli** [comment]
Not sure what's caused the build to break here - possibly some kind of config change that's been latent for a while since people haven't been committing to this repo.

**Comment by twhaples** [comment]
Looks like it's throwing a
```
NoMethodError: undefined method `spec' for nil:NilClass
```
on Ruby versions < 2. Huh.

File: .travis.yml [hunk 1]
```diff
@@ -9,3 +9,5 @@ notifications:
     recipients:
       - grey@gocardless.com
 script: bundle exec rspec spec
+before_install:
+  - gem install bundler
```
Comment (twhaples) [inline .travis.yml]: per  rubygems/rubygems#1419

**Comment by KushalP** [comment]
👍

**Pull request merged**